app/src/main/java/org/systers/mentorship/view/activities/MemberProfileActivity.kt

```diff
@@ -34,12 +37,17 @@ class MemberProfileActivity : BaseActivity() {
         supportActionBar?.title = getString(R.string.member_profile)
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
 
+        btnSendRequest.isEnabled=false
+
```

Comment:
Can you please remove the unnecessary space here..?

Reply:
+1